index.html

```diff
@@ -1,13 +1,221 @@
+                </div>
+            </div>
+
+    </section>
+    <section class="about-me">
+            <article class="about-myself">
+                <h1 style="margin: 0;">About me</h1>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience!
+                    If
+                    you
+                    like
+                    what you see and have a project you need coded,
+                    don't hestiate to contact me.
+                </p>
+                   <button>
+                            Get My Resume
+                   </button>
+            <img src="./img/illustration 1- about me-mobile.svg" alt="illustration-bottom-left">
+
+                   <img src="./img/illustration 2-about me mobile.svg" alt="illustration-bottom-left">
+            </article>
+
+    </section>
+
+    <section class="language">
+        <div class="card-holder-container">
+            <div class="holder-image">
+                <img src="./img/icon-languages.svg" alt="icon languages">
+            </div>
+            <h2 class="holder-title">
+                Languages
+            </h2>
+            <ul class="card-tags-container">
+                <li class="tags-item">React</li>
+                <li class="tags-item">Rubi on rails</li>
+                <li class="tags-item">css</li>
+                <li class="tags-item">JavaScript</li>
+                <li class="tags-item">html
+                <li>
+            </ul>
+        </div>
+
+        <div class="card-holder-container">
+            <div class="holder-image">
+                <img src="./img/icon-frameworks.svg" alt="icon frameworks">
+            </div>
+            <h2 class="holder-title">
+                Frameworks
+            </h2>
+            <div class="card-tags-container">
+                <div class="tags-item">Tailwind</div>
+                <div class="tags-item">Bootstrap</div>
+                <div class="tags-item">Selenium</div>
+                <div class="tags-item">RSpec</div>
+                <div class="tags-item">GraphQL</div>
+            </div>
+
+        </div>
+
+        <div class="card-holder-container">
+            <div class="holder-image">
+                <img src="./img/icon-skills.svg" alt="icon skills">
+            </div>
+            <h2 class="holder-title">
+                Skills
+            </h2>
+            <div class="card-tags-container">
+                <div class="tags-item">Codekit</div>
+                <div class="tags-item">GitHub</div>
+                <div class="tags-item">CodePen</div>
+                <div class="tags-item">GitLab</div>
+                <div class="tags-item">Terminal</div>
+            </div>
+        </div>
+
+    </section>
+
+    <h1 class="last-title-container">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get<br> in touch.
+    </h1>
```

Comment:
[ Required ] Please consider wrapping this text in a `h3` tag instead of an `h1` tag, this page already has an `h1` tag thus the most ideal tag to wrap it is another `hx` tag. Also please consider adding the `left` and `right` margins as shown in the Figma template below

## Figma template
<img src="https://user-images.githubusercontent.com/16392046/199610033-2240850b-1793-482b-813d-2e984fab1be2.png" width="300px" />

## You implementation
<img src="https://user-images.githubusercontent.com/16392046/199610219-07ab0656-1aa6-400b-ae3b-6b983625e1dc.png" width="300px" />